content/applications/marketing/marketing_automation/target_audience.rst

```diff
@@ -1,123 +1,123 @@
-reveals a drop-down menu. From this drop-down menu, select :guilabel:`contains`.
+- :guilabel:`Target`: defines the model of the target audience for the campaign. The assigned model
+  determines the fields that are available throughout the campaign, including the fields available
+  in the :guilabel:`Filter` section, and in dynamic placeholders.
+- :guilabel:`Save as Favorite Filter`: saves the current :guilabel:`Filter` for future use with the
+  current :guilabel:`Target` model, and can be managed from the :menuselection:`Marketing Automation
+  app --> Configuration --> Favorite Filters` menu.
+- :guilabel:`Unicity based on`: specifies the :guilabel:`Target` model field at which duplicates
+  should be avoided. Traditionally, the :guilabel:`Email` field is used, but any available field can
+  be used.
+- :guilabel:`Filter`: contains an interactive form to refine the target audience on a specific
+  subset of the :guilabel:`Target` model, detailed in the
+  :ref:`marketing_automation/defining-filters` section.
+- :guilabel:`Include archived`: allows or disallows the inclusion of archived records in the target
+  audience.
 
-In the third (and final) empty portion of the filter equation, type in the name of the event(s) that
-Odoo should consider for this campaign filter.
+.. tip::
+   A :guilabel:`Responsible` user can be assigned to the campaign by activating
+   :ref:`developer-mode`.
 
-Then, add another node to this filter by clicking the :guilabel:`➕ (plus sign)` icon to the right of
-the equation.
+.. note::
+   Each activity in a campaign's workflow can target a subset of the target audience, see the
+   :doc:`workflow_activities` documentation for more information.
 
-The second node will focus on targeting this campaign to attendees who purchase a specific type of
-ticket to the aforementioned event(s) mentioned in the first filter equation.
+.. _marketing_automation/defining-filters:
 
-To do that, select the default :guilabel:`ID` from the first portion of the second filter equation,
-and scroll down (or search for) :guilabel:`Event Ticket` from the field drop-down menu. Then, in
-that same drop-down menu, select :guilabel:`Name`.
+Defining filters
+================
 
-Once again, click the default :guilabel:`🟰 (equal sign)` icon in the second portion of the filter
-equation, and select :guilabel:`contains`.
+The default campaign :guilabel:`Filter` configuration is set to :guilabel:`Match all records`,
+indicating that the campaign is targeting **all** records of the :guilabel:`Target` model.
 
-Lastly, in the third (and final) portion of the second filter equation, which is blank, type in the
-name of the ticket type that should be used for the filter. In this case, :guilabel:`Standard` is
-the name of the event ticket type for this sample filter.
+To refine the :guilabel:`Filter` rules of a campaign, click the :guilabel:`➕ Add condition` button
+to reveal a new filter rule row.
```

Comment:
break up the [noun stack](https://www.enago.com/academy/noun-stacks-why-you-should-avoid-them-in-scientific-writing/)
```suggestion
to reveal a new row with configurable rule parameters.
```